18/README.md

```diff
@@ -2,211 +2,158 @@
-    {
-      "amount": "10",
-      "condition": {
-        "details": {
-          "public_key": "Election Address",
-          "type": "ed25519-sha-256"
-        },
-        "uri": "ni:///sha-256;b...123?fpt=ed25519-sha-256&cost=131072"
-      },
-      "public_keys": [
-        "Election Address"
-      ]
-    }
-  ],
-  "version": "2.0"
-}
-```
+#### Valid Election
+An Election proposal is a transaction representing the matter of change. The _Initiator_ ![v_it][eq_v_it], issues a create election transaction ![][EC_t]. The election MUST contain outputs  such that each output is populated with the public key of the validator ![][eq_v_k] s.t. ![][v_k_in_V_t], and an amount equal to the current power ![][eq_p(v_k, t)]  of the validator.
+
+#### Voting
+Once ![][EC_t] is committed in a block, the election starts. Independently and asynchronously, each validator may spend its vote tokens (referred as ![][T_k]) to the election address ![][EC_t_addr] to show agreement on the matter of change. The Election Address ![][EC_t_addr] is the `id` of the transaction ![][EC_t].
+
+NOTE: Once the vote tokens ![][T_k] have been transferred to the election address ![][EC_t_addr] it is not possible to transfer it again, because the private key is not known.
+
+#### Concluding Election
+At time ![][t_n] let,
+- Validator set be denoted by ![][V_t_n] s.t. ![][vi_in_V_tn]
+- ![][vm_in_Vm], where ![][V_m] denotes the set of public keys who voted for election ![][EC_t]
+- ![][T_new] be the newly received vote token
+
+
+##### Constrained approach (Approach 1)
+In the constrained approach any change to the validator set invalidates all the elections which were initiated with a different validator set.
+
+If below conditions hold true then the election is concluded and the proposed change is applied,
+1. ![][V_tn_equals_V_t]
+2. ![][constrain_condition1] where ![][v_k_in_V_t] and ![][T_k] denotes the vote tokens received at ![][EC_t_addr] prior the the current token
+3. ![][constrain_condition2]
+
+
+##### Generalized approach (Approach 2)
+The generalized constraints can tolerate a certain degree of change to the validator set.
+
+If below conditions hold then the election is concluded and the proposed change is applied,
+
+1. ![][general_condition1]
+2. ![][constrain_condition1] where ![][v_k_in_V_t] and ![][T_k] denotes the vote tokens received at ![][EC_t_addr] prior the the current token
+3. ![][constrain_condition2]
+
+The above constraints state that if the validators with which an election ![][EC_t] was initiated still hold super-majority then the election can be concluded.
+
+##### Approach 1 vs Approach 2
+
+Approach 1 is easier to comprehend and explain because of how constrained it is.
+
+Approach 2 has two benefits over the Approach 1.
+
+1. Elections can have intersections. In other words, the following is possible:
+
+```election 1 creation height < election 2 creation height < election 1 conclusion height < election 2 conclusion height```
```

Comment:
I was not right about the intersections - in case 2 elections do not update the validator set, they can interleave even in the first approach. So, the second approach is just about surviving validator set updates.